app/views/devise/shared/_links.html.haml

```diff
@@ -9,3 +9,7 @@
 - if devise_mapping.lockable? && resource_class.unlock_strategy_enabled?(:email) && controller_name != 'unlocks'
   = link_to "Didn't receive unlock instructions?", new_unlock_path(resource_name)
   %br
+
+- if devise_mapping.ichain_registerable?
+  = link_to "Sign up", new_ichain_registration_path(resource_name)
```

Comment:
@differentreality It's a helper in the `devise_ichain_authenticatable` gem that is used in their views. It seems to be used in one other place:
```
app/views/layouts/_navigation.html.haml
47:            %li{:class=> "#{active_nav_li(new_ichain_registration_path('user'))}"}
48:              = link_to(new_ichain_registration_path('user')) do
```

Should I update all usages of `new_ichain_registration_path` to `new_user_ichain_registration_path`?